docs/getting-started/install-endpoint.asciidoc

```diff
@@ -30,8 +30,44 @@ image::images/install-endpoint/endpoint-cloud-sec-integrations-page.png[Search r
 image::images/install-endpoint/endpoint-cloud-security-configuration.png[Add {elastic-defend} integration page,75%]
 +
 . Configure the {elastic-defend} integration with an **Integration name** and optional **Description**.
+. Select the type of environment you want to protect, either *Traditional Endpoints* or *Cloud Workloads*.
+. Next, select a configuration preset. Each preset comes with different default settings for {agent} — you can further customize these later. To learn more, refer to {fleet-guide}/agent-policy.html[{agent} policies].
+
+[cols="2"]
+|===
+|*Traditional Endpoint presets*
+| All Traditional Endpoint presets have the following preventions enabled by default: machine learning malware, ransomware, memory threat, malicious behavior, and credential theft.
+
+*Next Generation Antivirus:*
+
+Events collected: Process
+
+*Essential Endpoint Detection & Response:*
+
+Events collected: Process, Network, File
+
+*Complete Endpoint Detection & Response:*
+
+Events collected: All
+
+|*Cloud Workloads presets*
+|Both cloud Workload presets are intended for monitoring cloud-based Linux hosts, and therefore have <<session-view,Session data>> collection enabled by default, which enriches process events. All preventions are disabled by default.
+
+*All events:*
+
+Events collected: Process, Network, File
+
+*Interactive only:*
+
+Events collected: Process, Network, File
+
+Event filter: filters out non-interactive sessions
```

Comment:
I kind of stumbled over this, because I was trying to compare the two options but at first glance they look the same; they collect the same events. It'd be more effective if this list focused more on the _differences_ between the options, and all the similarities can go in the lead-in above. So roughly:

> Both cloud workload presets have session data collection enabled, preventions disabled, and collect process, network, and file events.
> -  **All events:** Monitors all system executions
> - **Interactive only:** Only monitors system interactions initiated by users

(I borrowed some of the UI copy for these options, which might help distinguish the features.)

Also, how is "event filter" being used here? [Event filters](https://www.elastic.co/guide/en/security/master/event-filters.html) are a specific artifact for Elastic Defend, so does this mean that the Interactive only option creates actual event filters on the Elastic Defend integration policy, or is this just saying in general that some events are not included?